### ISSUE TITLE
shelltile confuses secondary screens in a 3 screen setup

### ISSUE THREAD

**Issue by mirko** [emasab/shelltile#4]
Hey,

I'm having an issue with a 3 screen setup (1 primary, 2 secondary screens).
ShellTile seems to confuse 2 of the 3 screens.
When dragging a window into the corner of the primary screen it gets placed into the corner on a secondary screen and vice versa.
Attached are 2 screenshots which show the issue.
The big screen is the primary screen.

Apart from that: Thanks a lot for that plugin! :)

Trying to place the Firefox- window into upper-left corner of big screen:
![screenshot from 2016-07-14 17-34-01](https://cloud.githubusercontent.com/assets/106558/16845436/3333d60c-49e9-11e6-85f3-6753b32e0af4.png)
Trying to place the Terminal-window into left half of small screen:
![screenshot from 2016-07-14 17-36-18](https://cloud.githubusercontent.com/assets/106558/16845618/d8ed7544-49e9-11e6-9a07-25c4ba84069e.png)

**Comment by johannesgunnar** [comment]
Having the same problem with a 2 screen setup on Ubuntu Gnome 16.04. When I try to tile on my right screen, the overlay and tiling happens on the left screen, and the other way around.

![screens](https://cloud.githubusercontent.com/assets/2924543/16944510/7b7382aa-4d90-11e6-94de-e7ed12cce081.png)

In my current setup the main screen (2) is on the right, and the secondary screen (1) is on the left.

Oddly enough, this was not the problem before I reorganized the setup on my (physical) desktop. But prior to this I had the secondary screen (1) on the right, and main screen (2) on the left. I tried switching back to that in the Display settings. And the shelltile bug went away.
